CMakeLists.txt

```diff
@@ -430,13 +435,15 @@ else()
   set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${syscalls_subdirs_txt})
 endif()
 
+get_property(COMPLETE_LIST_SYSCALL_DIR GLOBAL PROPERTY SYSCALL_INCLUDE_DIRECTORY)
+
 add_custom_command(
   OUTPUT
   ${syscalls_json}
   COMMAND
   ${PYTHON_EXECUTABLE}
   ${ZEPHYR_BASE}/scripts/parse_syscalls.py
-  --include          ${ZEPHYR_BASE}/include        # Read files from this dir
```

Comment:
Per my change suggestion on muliple --include below, leave this here so we always scan the base include directory to pull in the main system syscalls

Reply:
yes with that change i should keep this line. 
But i still need to expand COMPLETE_LIST_SYSCALL_DIR to include the required directories.